includes/WikiPage.php

```diff
@@ -1907,7 +1907,7 @@ public function doUpdateRestrictions( array $limit, array $expiry, &$cascade, $r
 		$protectDescription = trim( $protectDescription );
 
 		if ( $id ) { # Protection of existing page
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)